client/python/gradio_client/serializing.py

```diff
@@ -515,6 +515,7 @@ def deserialize(
 
 COMPONENT_MAPPING: dict[str, type] = {
     "textbox": StringSerializable,
+    "multimodaltextbox": StringSerializable,
```

Comment:
I don't think its `StringSerialable` is correct. We'll need to create a new class that can take in a dict with string text and dict file, and serialize it properly.